addons/l10n_in_edi/tests/test_edi_json.py

```diff
@@ -265,3 +266,70 @@ def test_edi_json(self):
         })
         json_value = self.env['account.edi.format']._l10n_in_edi_generate_invoice_json(self.invoice_negative_more_than_max_line)
         self.assertDictEqual(json_value, expected, "Indian EDI with negative value more than max line sent json value is not matched")
+
+    def test_edi_json_equal_sgst_cgst(self):
+        """
+            In price include tax some time we have issue where sgst cgst tax amount is not equal
+            So we create rounding tax with 0.01
+        """
+        round_off_tax = self.env['account.tax'].create({
+            'name': 'Round Off',
+            'type_tax_use': 'sale',
+            'amount_type': 'percent',
+            'amount': 0.01,
```

Comment:
```suggestion
            'amount': 0,
```
@jva-odoo Could it work if we put this at 0 (or as a fixed amount_type?)

Reply:
@jco-odoo no it's not working.
and fix amount is multiplied by qty so if we put 0.01 as fix then it gives the wrong value when qty is more than 1